_posts/2023-02-07-Announcing-Data-Prepper-2.1.0.md

```diff
@@ -0,0 +1,256 @@
+which works well with tools such as [FluentBit](https://fluentbit.io). If you would like to deploy the 
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) on your applications 
+and not require any other sidecars, you can now send logs to Data Prepper, which will route them to OpenSearch.
+
+
+## OpenSearch sink improvements
+
+Running Data Prepper as an ingestion pipeline in front of OpenSearch is an important capability for Data Prepper. This 
+release adds hew enhancements to Data Prepper’s OpenSearch sink to make it even more useful with OpenSearch.
+
+Data Prepper now supports routing traces and logs to different indexes dynamically. To achieve this, you can define an 
+index name using a format string which can include properties from different events. With this capability, Data Prepper 
+can support an arbitrary number of indexes using a single sink. For example, if you have logs with different 
+application identifiers, you can route events to an index specifically for each application.
+
+Data Prepper also now also allows routing documents to specific OpenSearch shards using the existing `routing` parameter 
+in OpenSearch. The new `routing_field` property on the OpenSearch sink will use properties from events to specify how to 
+route within OpenSearch. Most users prefer allowing OpenSearch to generate document IDs and choose the shard 
+from that. However, Data Prepper now allows you to specify these values explicitly.
+
+Going hand-in-hand with the routing field, Data Prepper now lets you specify a complex field to 
+use when specifying a document ID. Previously, you could only copy a value to a field in the root of 
+the event to use it as a document ID.
+
+## Type conversion
+
+Data Prepper now supports a new 
+[convert entry](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/mutate-event-processors#convertentryprocessor) 
+processor as part of 
+[mutate processors](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/mutate-event-processors#mutate-event-processors) 
+to convert a value from one type to another. This can be particularly useful in conditional expressions when using conditional routing.
+
+For example, using the following processor in a pipeline lets you convert the value of `status` key to `integer`: 
+
+```
+...
+processor:
+  - convert_entry_type:
+      key: "status"
+      type: "integer"
+...
+```
+
+## Other improvements
+
+In addition to the new features already described, Data Prepper 2.1.0 has several other important improvements:
+
+* Data Prepper pipelines with OpenSearch sinks now wait for OpenSearch to start during pipeline initialization if the sink is not available instead of failing.
+*  OpenSearch sink now allows you to load index mapping and ISM template files from an S3 bucket. You can configure the S3 object by providing its S3 URI in existing configuration option to load them.
```

Comment:
```suggestion
*  OpenSearch sinks now allow you to load index mapping and Index State Management (ISM) template files from an Amazon Simple Storage Service (Amazon S3) bucket. You can configure the S3 object by providing its S3 URI in existing configuration option to load them.
```

Reply:
The last sentence is a bit confusingly written, is missing articles, and has an unclear subject. Please revise for simplicity/clarity.

Reply:
I'll just remove it. I don't think we need it in the release blog post.